pages/pipelines/block_step.md

```diff
@@ -11,20 +11,22 @@ Once all steps before the block have completed, the pipeline will pause and wait
 
 ```yml
 steps:
-  - block: "\:rocket\: Release!"
+  - block: "\:rocket\: Are we ready?"
 ```
 {: codeblock-file="pipeline.yml"}
 
-<%= image "block-step.png", width: 944/2, height: 364/2, alt: "Screenshot of a basic block step" %>
+<%= image "block-step-button.png", alt: "Screenshot of button to press to unblock a block step" %>
```

Comment:
As mentioned offline, I was thinking that we might need the widths + heights for these images but now that I see them full width I think it's probably OK – assuming the image is at least double the container column size.

Reply:
Yeah they are exported at 2x 1080